root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base

```diff
@@ -18,7 +18,7 @@
     </IfModule>
 
     <FilesMatch \.php$>
-      SetHandler "proxy:fcgi://127.0.0.1:'.$php_port.'"
```

Comment:
The `$php_port` variable is not used anymore, together with the `TCPPorts` prop. Let's remove them!

Reply:
done...sorry, I missed it